SpringProject/src/main/java/com/gradproject2019/userConference/service/UserConferenceServiceImpl.java

```diff
@@ -38,11 +45,32 @@ public UserConferenceResponseDto saveInterest(UUID token, UserConferenceRequestD
         }
     }
 
+    @Override
+    public List<ConferenceResponseDto> getConferenceByUserId(UUID token, Long userId) {
+        List<ConferenceResponseDto> conferenceResponseDtos = new ArrayList<>();
+
+        checkUserAuthorised(token);
+        List<UserConference> userConferences = userConferenceRepository.findByUserId(userId);
+
+        for(UserConference userConference: userConferences) {
+            conferenceResponseDtos.add(conferenceService.getConferenceById(userConference.getConferenceId()));
+        }
+        return conferenceResponseDtos;
+    }
+
     private void checkUserAuthorised(UUID token) {
         try {
             authService.checkTokenExists(token);
         } catch (TokenNotFoundException e) {
             throw new UserUnauthorisedException();
         }
     }
+
+    @Override
+    public boolean checkUserIsInterestedInConference(Long conferenceId){
+        if (userConferenceRepository.findByConferenceId(conferenceId).size() == 0) {
```

Comment:
This method could probably be better served by a "doesInterestExist" method in your repository which would return a boolean if a conference does exist :)

Reply:
This is a good point,  as this method is actually more relevant to the ticket for de-expressing interest (and is reliant on the delete endpoint) I am going to remove it from this PR.

Reply:
Also what does `if <condition> then false else true` tell you about the condition?
You can just return `!<condition>`